INSTALL.md

```diff
@@ -234,10 +234,7 @@ Use the following command to install OpenSSL.
 
 By default, OpenSSL will be installed to
 
-    SYS$COMMON:[OPENSSL-'version'...]
-
-where 'version' is the OpenSSL version number with underscores instead
-of periods.
```

Comment:
> `SYS$COMMON:[OPENSSL-'version'...]` -> `SYS$COMMON:[OPENSSL]` 

Are you really just clarifying something here or are you documenting a change of behaviour?

Reply:
I'm correcting a documentation error.  descrip.mms really does have `SYS$COMMON:[OPENSSL]` as the default install top.  What I've removed was something that was true long ago, before we changed things across the board to have the version embedded into the shared library names.